projects/plugins/jetpack/extensions/shared/memberships/settings.js

```diff
@@ -315,7 +315,7 @@ export function NewsletterEmailDocumentSettings() {
 	const isSendEmailEnabled = useSelect( select => {
 		const meta = select( editorStore ).getEditedPostAttribute( 'meta' );
 		// Meta value is negated, "don't send", but toggle is truthy when enabled "send"
```

Comment:
I'm not sure whether this comment is still necessary. Now that the toggle control value is a string, the code is more self-documenting IMO.